templates/base.html

```diff
@@ -23,6 +23,7 @@
 
     <!-- JS -->
     <script src="/js/jquery/jquery.js"></script>
+    <script src="js/main/responsive.js"></script>
```

Comment:
Import should be relative to domain root.
Add "/" in front of the string ("/js/...")